packages/core/src/diff/changes/argument.ts

```diff
@@ -39,12 +40,19 @@ export function fieldArgumentDefaultChanged(
     type: ChangeType.FieldArgumentDefaultChanged,
     message:
       typeof oldArg.defaultValue === 'undefined'
-        ? `Default value '${newArg.defaultValue}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
-        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${oldArg.defaultValue}' to '${newArg.defaultValue}'`,
+        ? `Default value '${safeString(newArg.defaultValue)}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
+        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${safeString(oldArg.defaultValue)}' to '${safeString(newArg.defaultValue)}'`,
     path: [type.name, field.name, oldArg.name].join('.'),
   };
 }
 
+function safeString(obj: any) {
```

Comment:
suggestion: Let's add a small unit test for `safeString` so we can verify it does what it's supposed to.

We should test the original issue (no `toString` method) and a few common scenarios like `undefined`, `null`, etc. The test should cover all three branches in the if statement -- `object == null`, `obj.toString === 'function'` and `obj.toString !== 'function'`